packages/vscode/package.json

```diff
@@ -69,6 +69,9 @@
     "dev": "tsup --watch src",
     "publish": "esno ./scripts/publish.ts"
   },
+  "dependencies": {
+    "pathe": "^0.3.7"
```

Comment:
This should go devDeps as VS Code is not a npm package.

Reply:
resolved